src/greeter/waylandkeyboardbackend.cpp

```diff
@@ -43,7 +43,7 @@ WaylandKeyboardBackend::~WaylandKeyboardBackend()
 {
 }
 
-QList<QObject *> parseRules(const QString &filename, int &current)
+QList<QObject *> parseRules(const QString &filename, [[maybe_unused]] int &current)
```

Comment:
Looks like a genuine bug. The current layout is not set by the wayland backend?

Reply:
I am not sure, this is a bit above my paygrade.
Any idea how to fix this?

Reply:
No idea. There is no compositor agnostic API for this. Maybe just

```
// FIXME
current = 0;
```